app/models/user.rb

```diff
@@ -50,38 +50,38 @@ def new_contributor
     return "<span class = 'label label-success'><i>New Contributor</i></span>".html_safe if Node.where(:uid => @uid).length === 1
   end
 
-  def create_drupal_user
+  def create_user
     self.bio ||= ''
-    if drupal_user.nil?
-      drupal_user = DrupalUser.new(name: username,
-                              pass: rand(100_000_000_000_000_000_000),
-                              mail: email,
-                              mode: 0,
-                              sort: 0,
-                              threshold: 0,
-                              theme: '',
-                              signature: '',
-                              signature_format: 0,
-                              created: DateTime.now.to_i,
-                              access: DateTime.now.to_i,
-                              login: DateTime.now.to_i,
-                              status: 1,
-                              timezone: nil,
-                              language: '',
-                              picture: '',
-                              init: '',
-                              data: nil,
-                              timezone_id: 0,
-                              timezone_name: '')
-      drupal_user.save!
-      self.id = drupal_user.uid
+    if user.nil?
+      	    user = User.new(name: username,
+                                    pass: rand(100_000_000_000_000_000_000),
```

Comment:
I guess here is the error with `pass` that is coming in the tests. I guess pass is an unknown attribute with User model. So adding pass attribute to User model might do the work.

Reply:
yeah my app/models/user.rb looks like that.
I ran rake `db:setup` and `rake db:migrate` which now shows me this error on localhost:
couldn't find file 'noty/lib/noty.css' with type 'text/css'

Reply:
Aha! This corresponds to this section, where a matching drupal_user is created:

https://github.com/publiclab/plots2/blob/bce1f1fb7847f5444474f26218284cfaf602d2c5/app/models/user.rb#L53-L81

But since we're deprecating drupal_user, we can delete that whole section -- we don't need to do it anymore. We can also remove the hook which triggers it:

https://github.com/publiclab/plots2/blob/bce1f1fb7847f5444474f26218284cfaf602d2c5/app/models/user.rb#L44

Make sense?

Reply:
I think so, let me delete those and see what happens

Reply:
Hmm right @jywarren, I guess deleting would be better and the removal of hooks if not done would be detected by tests. So it would work fine. So @Neidley you can remove this function in my opinion.

Reply:
Do I need to make changes locally, or are we able to make changes within this PR and see if it works from here?
(Still new to the capabilities/complexities of PRs) and if changes can be made by you guys here how do I pull them back to my local branch to run code locally?

Reply:
You should do the changes locally and push it here and if there is any conflicts shown up in the PR. Then this means you have to rebase it as you have done today.

Reply:
So I'm deleting the `create_user` function from plots2/app/models/user.rb? (that was previously create_drupal_user function) and see what happens? Don't users need to still be created?